src/lib/etl-glue.ts

```diff
@@ -176,6 +177,7 @@ export class ETLByGlue extends Construct {
       },
     });
     const outputPrefix = `${props.s3Prefix ?? ''}processed-data/`;
+    this.preprocessingJob_id_cols = 'card1,card2,card3,card4,card5,card6,ProductCD,addr1,addr2,P_emaildomain,R_emaildomain';
```

Comment:
I prefer to move this constant to parent stack, also rename it to `datasetIdCols`